src/cli/index.ts

```diff
@@ -222,45 +222,54 @@ async function main(argv: string[]): Promise<void> {
       switch (type) {
         case 'SetDefaultOp':
           console.log(`  ${bold(`Add missing \`DEFAULT\` constraints to the database`)}`)
-          console.log(`  ${gray(`https://pris.ly/d/schema-incompatibilities#default-values-arent-represented-in-database`)}`)
+          console.log(`  ${gray(underline(`https://pris.ly/d/schema-incompatibilities#default-values-arent-represented-in-database`))}`)
           console.log()
           console.log('    ' + queries.join('\n    '))
           console.log()
+          console.log()
           break
         case 'SetCreatedAtOp':
-          console.log(`  ${bold(`Replicate \`@createdAt\` behavior in Prisma 2.0`)}`)
-          console.log(`  ${gray(`https://pris.ly/d/schema-incompatibilities#createdat-isnt-represented-in-database`)}`)
+          console.log(`  ${bold(`Replicate \`@createdAt\` behavior in Prisma 2`)}`)
+          console.log(`  ${gray(underline(`https://pris.ly/d/schema-incompatibilities#createdat-isnt-represented-in-database`))}`)
           console.log()
           console.log('    ' + queries.join('\n    '))
           console.log()
+          console.log()
           break
         case 'AddUniqueConstraintOp':
           console.log(`  ${bold(`Fix 1-1 relations by adding \`UNIQUE\` constraints`)}`)
-          console.log(`  ${gray(`https://pris.ly/d/schema-incompatibilities#inline-1-1-relations-are-recognized-as-1-n-missing-unique-constraint`)}`)
+          console.log(`  ${gray(underline(`https://pris.ly/d/schema-incompatibilities#inline-1-1-relations-are-recognized-as-1-n-missing-unique-constraint`))}`)
           console.log()
           console.log('    ' + queries.join('\n    '))
           console.log()
+          console.log()
           break
         case 'SetJsonTypeOp':
           console.log(`  ${bold(`Fixing columns with JSON data types`)}`)
-          console.log(`  ${gray(`https://pris.ly/d/schema-incompatibilities##json-type-is-represented-as-text-in-database`)}`)
+          console.log(`  ${gray(underline(`https://pris.ly/d/schema-incompatibilities##json-type-is-represented-as-text-in-database`))}`)
           console.log()
           console.log('    ' + queries.join('\n    '))
           console.log()
+          console.log()
           break
       }
     }
 
     console.log(
       redent(`
-      If you've made all the SQL changes you're comfortable with,
-      you can skip to the end where the Upgrade CLI makes some final
-      adjustments to your Prisma 2.0 schema.
-
-      Otherwise the next steps are to:
+      If you executed one or more of the above SQL statements against your database, 
+      please run the following two commands: 
 
-        1. Run ${green(`\`prisma introspect\``)} again to refresh your Prisma 2.0 schema.
+        1. Run ${green(`\`prisma introspect\``)} again to refresh your Prisma 2 schema.
```

Comment:
Default we also suggest `npx` here, right?
```suggestion
        1. Run ${green(`\`npx prisma introspect\``)} again to refresh your Prisma 2 schema.
```